Sources/Logging/Logging.swift

```diff
@@ -39,20 +39,44 @@ import WASILibc
 /// logger.info("Hello World!")
 /// ```
 public struct Logger {
-    @usableFromInline
-    var handler: LogHandler
+    /// A private property to hold the boxed `LogHandler`.
+    private var _handler: Storage
```

Comment:
Let's rename this and also add the inline annotation back. This is going to be important for performance reasons.
```suggestion
    @usableFromInline
    private var _storage: Storage
```

Reply:
`@usableFromInLine` can only be applied to internal or declaration, should I change `_storage` from private?

Reply:
@ayushi2103 Yes please! You already named it correctly, the `_` signifies "this is internal but should be private". And `public var _foo` means "this is public but should be private". 

Especially for `internal var _foo` it's almost always because of `@inlinableFromPublic`.